src/commands/local-store/post-local-store-paranet-validate-command.js

```diff
@@ -0,0 +1,84 @@
+import { setTimeout } from 'timers/promises';
+
+import { LOCAL_STORE_TYPES, ERROR_TYPE, OPERATION_ID_STATUS } from '../../constants/constants.js';
+import Command from '../command.js';
+
+class PostLocalStoreParanetValidateCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.operationIdService = ctx.operationIdService;
+        this.paranetService = ctx.paranetService;
+        this.blockchainModuleManager = ctx.blockchainModuleManager;
+        this.commandExecutor = ctx.commandExecutor;
+        this.ualService = ctx.ualService;
+        this.repositoryModuleManager = ctx.repositoryModuleManager;
+
+        this.errorType = ERROR_TYPE.VALIDATE_ASSET_ERROR;
+    }
+
+    async execute(command) {
+        const {
+            blockchain,
+            contract,
+            tokenId,
+            storeType = LOCAL_STORE_TYPES.TRIPLE,
+            paranetId,
+        } = command.data;
+
+        const operationId = await this.operationIdService.generateOperationId(
+            OPERATION_ID_STATUS.VALIDATE_ASSET_START,
+        );
+
+        try {
+            if (storeType === LOCAL_STORE_TYPES.TRIPLE_PARANET) {
+                await setTimeout(5 * 60 * 1000);
+                const assetUal = this.ualService.deriveUAL(blockchain, contract, tokenId);
+
+                this.logger.info(
+                    `[Post Local-Store] Checking if asset: ${assetUal} is part of paranet: ${paranetId}, operation id: ${operationId}.`,
+                );
+                const knowledgeAssetId = this.paranetService.constructKnowledgeAssetId(
+                    blockchain,
+                    contract,
+                    tokenId,
+                );
+
+                const isKnowledgeAssetRegistered =
+                    await this.blockchainModuleManager.isKnowledgeAssetRegistered(
+                        blockchain,
+                        paranetId,
+                        knowledgeAssetId,
+                    );
+
+                if (!isKnowledgeAssetRegistered) {
+                    this.logger.info(
+                        `[Post Local-Store] Asset: ${assetUal} is NOT part of paranet: ${paranetId}, operation id: ${operationId}. Deleting it from paranet operational table.`,
+                    );
+
+                    await this.repositoryModuleManager.deleteParanetSyncedAssetRecord(assetUal);
```

Comment:
What about counts

Reply:
We don't use count anymore, but I can add it